pages/osx/pod.md

```diff
@@ -0,0 +1,31 @@
+# pod
+
+> Dependency manager for Swift and Objective-C Cocoa projects.
+
+- Create a Podfile for the current project with the default contents:
+
+`pod init`
+
+- Download all pod defined in Podfile and create an Xcode Pods library project in `./Pods`:
+
+`pod install`
+
+- Download a new pod for the current project and add it to the Podfile:
+
+`pod install {{pod_name}}`
```

Comment:
Are you sure this option works ? I don't see one in the docs page. As far as I am seeing, the only way to add a new pod is to add it to the Podfile and run `pod install`.